superset/assets/visualizations/markup.js

```diff
@@ -23,9 +24,11 @@ function markupWidget(slice, payload) {
     <iframe id="${iframeId}"
       frameborder="0"
       height="${slice.height()}"
-      sandbox="allow-scripts">
+      sandbox="allow-same-origin allow-scripts allow-top-navigation allow-popups">
     </iframe>`);
-  $('#' + iframeId)[0].srcdoc = html;
+
+  const iframe = $('#' + iframeId)[0];
```

Comment:
indentation is off and while we are at it we may just use a plain dom js function instead of jquery

Reply:
jquery is now removed